### ISSUE TITLE
what is the use of adding a fc layer when reloading the pretrained model

### ISSUE THREAD

**Issue by alicefqy** [last-one/Pytorch_Realtime_Multi-Person_Pose_Estimation#12]
With some error with gpu memory, I've got to relance the train part.
In the loading part, a new fc layer is added. What does it mean?  
```
# state_dict = torch.load(args.pretrained)['state_dict']
# from collections import OrderedDict
# new_state_dict = OrderedDict()
# for k, v in state_dict.items():
        # name = k[7:]
        # new_state_dict[name] = v
# model.load_state_dict(new_state_dict)
# model.fc = nn.Linear(2048, 80)
```

**Comment by last-one** [comment]
Sorry. I forgot to remove it. It's redundant code.

**Comment by qy-feng** [comment]
Thanks for your response.
By the way, do you have any script to evaluate the trained model on a test set to get mAP?

**Comment by pingjunLi** [comment]
yea,I have same question ,as we used the caffe version.
